src/devices/machine/t10mmc.cpp

```diff
@@ -622,8 +626,8 @@ void t10mmc::ExecCommand()
 			auto track_type = m_image->get_track_type(trk);
 
 			// If there's a transition between CD data and CD audio anywhere in the requested range then return an error
-			if ((last_track_type == cdrom_file::CD_TRACK_AUDIO && track_type != cdrom_file::CD_TRACK_AUDIO)
-			|| (last_track_type != cdrom_file::CD_TRACK_AUDIO && track_type == cdrom_file::CD_TRACK_AUDIO))
+			if (last_track_type != -1 && ((last_track_type == cdrom_file::CD_TRACK_AUDIO && track_type != cdrom_file::CD_TRACK_AUDIO)
+			|| (last_track_type != cdrom_file::CD_TRACK_AUDIO && track_type == cdrom_file::CD_TRACK_AUDIO)))
```

Comment:
When no tracks have been seen yet (`last_track_type` = -1) and an audio track is encountered as the first track, it would throw an error. Only applies to the READ CD command.